docs/concepts/models/external_models.md

```diff
@@ -18,7 +18,7 @@ The querying model's [`kind`](./model_kinds.md), [`cron`](./overview.md#cron), a
 
 ## Generating an external models schema file
 
-External models are defined in the `schema.yaml` file in the SQLMesh project's root folder.
+External models can be defined in the `schema.yaml` file in the SQLMesh project's root folder.
```

Comment:
Can we also update this line: https://github.com/TobikoData/sqlmesh/blob/054f665e8e62eb7296bcab46d1db52167276177e/docs/concepts/models/external_models.md?plain=1#L41

to include something like:
```
Alternatively, additional external models can also be defined in the [external_models/](#using-the--external_models--directory) folder.
```
(Haven't tested the link)

Reply:
Sure, have updated it